pkgs/tools/misc/plocate/default.nix

```diff
@@ -8,20 +9,22 @@
 , liburing
 , zstd
 }:
-
-stdenv.mkDerivation rec {
+let
+  dbfile = lib.attrByPath [ "locate" "dbfile" ] "/var/cache/locatedb" config;
```

Comment:
I wouldn't mind if we didn't support this. It would make things a bit easier.

Reply:
Not supporting it leads to special cases in other places.
`/var/lib/plocate` needs to be made writable and the updatedb-script needs special casing for not adding `--output` argument. And we should also add a warning in `services.locate.output` is set to match the other warning behavior. 
To me it looks like we either need to complicate the locate service even further, or keep in locate to the plocate package.

I am open to both, but supporting it in the plocate package will isolate the complexity.

Reply:
I fixed up the patch somewhat and got it upstreamed, it will be part of the next plocate release.
That should reduce the complexity enough for this to be OK I think.
@SuperSandro2000